backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -164,14 +166,14 @@ public Organization updateOrganization(String name) {
         return _repo.save(org);
     }
 
-    public Facility createFacility(String testingFacilityName, String cliaNumber, StreetAddress facilityAddress, String phone,
+    public Facility createFacility(String testingFacilityName, String cliaNumber, StreetAddress facilityAddress, String phone, String email,
```

Comment:
I sort of want to add a new wrapper class for phone and e-mail, since they always (?) appear together. This would actually not be very useful but would help break up the long String, String, String argument lists. 😄